src/custom/utils/xdai/hack.ts

```diff
@@ -1,6 +1,7 @@
 // this file essentially provides all the overrides employed in uniswap-xdai-sdk fork
 // + logic for chainId switch to/from xDAI
```

Comment:
Could this hack be removed maybe, now that Uni supports other chains?

For another PR, or course.